src/script.js

```diff
@@ -40,17 +15,67 @@ createTheModal = () => {
       globalModal.style.display = "none";
     }
   }
+  console.log('window', window);
 }
 
-let modalInformation = document.getElementById("modalInformation")
-console.log('modalInformation', modalInformation);
+globalInfo = () => {
 
+  modal.style.display = "block";
```

Comment:
so it looks like you're creating your own custom version of a modal here, which I would _highly_ recommend you not do. Bootstrap takes care of this for, and there really is no sense in reinventing the wheel. Most importantly though, whoever comes to this after you is going to have to learn everything about your custom implementation and any/all of its quirks.

most of the interaction you're trying to custom create is handled by the bootstrap modal component: https://getbootstrap.com/docs/4.0/components/modal/